Libraries/Utilities/Appearance.js

```diff
@@ -85,6 +85,22 @@ module.exports = {
     return nativeColorScheme;
   },
 
+  setColorScheme(colorScheme: string): void {
+    const nativeColorScheme: ?string =
+      colorScheme == null ? 'unspecified' : colorScheme;
```

Comment:
We should either:
1. Omit the null-check because colorScheme can never be null
2. Mark the colorScheme parameter as nullable